garak/buffs/paraphrase.py

```diff
@@ -39,7 +39,9 @@ def _load_model(self):
         self.para_model = PegasusForConditionalGeneration.from_pretrained(
             self.para_model_name
         ).to(self.device)
-        self.tokenizer = PegasusTokenizer.from_pretrained(self.para_model_name)
+        self.tokenizer = PegasusTokenizer.from_pretrained(
+            self.para_model_name, trust_remote_code=False
```

Comment:
Should this be possible to inject via config? Consider adding this to `DEFAULT_PARAMS["hf_args"]` for this class.
```suggestion
            self.para_model_name, trust_remote_code=self.hg_args["trust_remote_code"]
```

Reply:
Excellent, yes, missed this, better route

Reply:
amended to default param+value